ecosystem/typescript/sdk_v2/src/utils/memoize.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * The global cache Map
+ */
+const cache = new Map<string, { value: any; timestamp: number }>();
```

Comment:
This could grow very large right?

We may want to think about things like cache configuration management by the user as well as if there is a way to turn it off.

Reply:
agree. we can introduce `clear` method (clear all cache / clear by key / etc).